assets/js/blocks/products-by-attribute/edit-mode.tsx

```diff
@@ -0,0 +1,78 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Button, Placeholder } from '@wordpress/components';
+import { category, Icon } from '@wordpress/icons';
+import ProductAttributeTermControl from '@woocommerce/editor-components/product-attribute-term-control';
+
+/**
+ * Internal dependencies
+ */
+import { Attributes } from './types';
+
+export interface Props {
```

Comment:
we can have this one extend Props and just add `isEditing/setIsEditing`

`export interface EditModeProps extends Props { ... }`